tests/i18n/test_extraction.py

```diff
@@ -604,13 +604,26 @@ def test_keep_pot_enabled(self):
 class MultipleLocaleExtractionTests(ExtractorTests):
     PO_FILE_PT = 'locale/pt/LC_MESSAGES/django.po'
     PO_FILE_DE = 'locale/de/LC_MESSAGES/django.po'
+    PO_FILE_KO = 'locale/ko/LC_MESSAGES/django.po'
     LOCALES = ['pt', 'de', 'ch']
 
     def test_multiple_locales(self):
         management.call_command('makemessages', locale=['pt', 'de'], verbosity=0)
         self.assertTrue(os.path.exists(self.PO_FILE_PT))
         self.assertTrue(os.path.exists(self.PO_FILE_DE))
 
+    def test_all_locales(self):
+        """
+        When the `locale` flag is absent, all dirs from the parent locale dir
+        are considered as language directories, except if the directory doesn't
+        start with two letters (which excludes __pycache__, .gitignore, etc.).
+        """
+        os.mkdir(os.path.join('locale', '_do_not_pick'))
+        # Excluding locales that do not compile
+        management.call_command('makemessages', exclude=['ja', 'es_AR'], verbosity=0)
+        self.assertTrue(os.path.exists(self.PO_FILE_KO))
+        self.assertFalse(os.path.exists('locale/_do_not_pick/LC_MESSAGES/django.po'))
```

Comment:
I wonder if something like `self.PO_FILE_KO.replace('/ko/', '_do_not_pick`)` would make that a bit more resilient to future changes. No strong feeling either way.